bin/_docker.sh

```diff
@@ -82,6 +84,13 @@ See https://github.com/docker/buildx/issues/59 for more details'
       fi
     fi
 
+    # Allow for specifying docker builder engine
+    # This is a great way to use k8s to build docker images on native hardware instead of emulated
+    # See https://docs.docker.com/build/drivers/kubernetes/ for an example
+    if [ "$DOCKER_BUILDER" ]; then
+            output_params+=" --builder=$DOCKER_BUILDER"
```

Comment:
Indentation is not very consistent in this file, but let's at least align it with the other `if`s in this function
```suggestion
      output_params+=" --builder=$DOCKER_BUILDER"
```